src/components/Person/Person.jsx

```diff
@@ -1 +1,54 @@
 // export const Person = ({ person }) => ();
+export const misha = {
+  name: 'Misha',
+  age: 37,
+  sex: 'm',
+  isMarried: true,
+  partnerName: 'Natasha',
+};
+
+export const olya = {
+  name: 'Olya',
+  sex: 'f',
+  isMarried: true,
+  partnerName: 'Maksym',
+};
+
+export const alex = {
+  name: 'Alex',
+  age: 25,
+  sex: 'm',
+  isMarried: false,
+};
+
+function partner(isMarried, sex, partnerName = undefined) {
+  if (isMarried) {
+    const personPartner = sex === 'm' ? 'wife' : 'husband';
+
+    return `${partnerName} is my ${personPartner}`;
+  }
+
+  return 'I am not married';
+}
+
+function years(personAge) {
+  if (personAge) {
+    return `I am ${personAge}`;
+  }
+
+  return '';
+}
+
+export const Person = ({ person }) => (
+  <section className="Person">
```

Comment:
1. it's a good practice to apply object destructing
2. create a variable I noticed above
```suggestion
export const Person = ({ person }) => {
  const {...} = person;
  
  const personPartner = sex === 'm' ? 'wife' : 'husband';
  
  return (
    <section className="Person">
```